CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ All notable changes to this project will be documented in this file.
 
 * New Api module which exposes (via the `ApiFacade`) the functions documentation of Phel (#551)
 * Rename command `phel compile` to `phel build` (#555)
+* Add new config parameter `ignore-when-compiling` (#557)
```

Comment:
Renamed to `ignore-when-building` https://github.com/phel-lang/phel-lang/commit/01c42f6a519a6dd464b9cafc893a3572455f1fe7